sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsDiskStore.scala

```diff
@@ -82,20 +86,25 @@ class RapidsDiskStore(diskBlockManager: RapidsDiskBlockManager)
       incoming: RapidsBuffer,
       path: File,
       append: Boolean,
-      stream: Cuda.Stream): (Long, Long) = {
+      stream: Cuda.Stream): (Long, Long, Long) = {
```

Comment:
Can we update the scaladocs to reflect what is returned now.

Reply:
updated